quad_name: 'hummingbird'
use_ekf_synchronization: False
control_freq_factor: 5

# Topic Names
imu_topic: "/hummingbird/imu"

# GPy Model
mpc_model_version: "SISO"
mpc_model_name: "789_789_20_100_mpc_gz"
mpc_model_type: "GPy"

mhe_type: "dynamic"
mhe_model_version: "SISO"
mhe_model_name: "678_789_50_100_mhe_gz"
mhe_model_type: "GPy"
